test.js

```diff
@@ -17,9 +17,18 @@ test('main', async t => {
 	await server.close();
 });
 
+test('support HTML input', async t => {
+	const {stdout} = await execa('./cli.js', [], {
+		input: '<h1>Unicorn</h1>',
+		encoding: 'buffer'
```

Comment:
Huh? It's a string.

Reply:
I'm confused, is this `encoding` for `stdout`?

https://github.com/sindresorhus/execa#encoding

Reply:
Oops, yes.